backend/src/main/scala/akka/viz/events/EventFiltering.scala

```diff
@@ -21,7 +24,13 @@ case object IsUserActor extends FilteringRule {
 case object AllowAll extends FilteringRule { override def apply(event: Received) = true }
 
 case class AllowedClasses(names: List[String]) extends FilteringRule {
-  @transient private val allowedClasses = names.map(name => getClass.getClassLoader.loadClass(name)).toSet
+  @transient private val allowedClasses = names.flatMap { name =>
+    val loaded: Try[Class[_]] = Try(getClass.getClassLoader.loadClass(name))
```

Comment:
I guess ClassLoader here might be different than the one in the application
Also my doubt is if it is safe to load any class, could that code pull something with static initialization?